vignettes/Workflow.Rmd

```diff
@@ -0,0 +1,479 @@
+---
+title: "Some Basic Analysis of ChIP-Seq Data"
+authors:
+  - name: "Deepayan Sarkar"
+  - name: "Robert Gentleman"
+  - name: "Michael Lawrence"
+  - name: "Zizhen Yao"
+  - name: "Oluwabukola Bamigbade" 
+    affiliation: "Vignette translation from Sweave to Rmarkdown / HTML"
+date:  "`r format(Sys.time(), '%B %d , %Y')`"
+vignette: >
+  %\VignetteIndexEntry{Some Basic Analysis of ChIP-Seq Data}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+  %\VignetteKeywords{chip-seq, sequencing, visualization}
+  %\VignettePackage{chipseq}
+  
+output:
+  BiocStyle::html_document
+---
+
+Our goal is to describe the use of Bioconductor software to perform some
+basic tasks in the analysis of ChIP-Seq data. We will use several
+functions in the as-yet-unreleased [chipseq]{.sans-serif} package, which
+provides convenient interfaces to other powerful packages such as
+[ShortRead]{.sans-serif} and [IRanges]{.sans-serif}. We will also use
+the [lattice]{.sans-serif} and [rtracklayer]{.sans-serif} packages for
```

Comment:
The patterns following `[chipseq]{.sans-serif}` are packages, so I will ask you to replace them with code to generate canonical links: https://bioconductor.org/packages/release/bioc/vignettes/BiocStyle/inst/doc/AuthoringRmdVignettes.html#4_Style_macros. Packages can be from Bioconductor, CRAN, or from somewhere else so you might have to search. You can use https://bioconductor.org/packages/release/BiocViews.html#___Software to search for Bioconductor packages.